txpool/txpool_test.go

```diff
@@ -20,6 +20,26 @@ var forks = &chain.Forks{
 	Istanbul:  chain.NewFork(0),
 }
 
+type mockStore struct {
+}
+
+func (m *mockStore) GetNonce(root types.Hash, addr types.Address) uint64 {
```

Comment:
Nitpick, but you can omit both of those:
```suggestion
func (m *mockStore) GetNonce(types.Hash, types.Address) uint64 {
```

Reply:
Fixed 👍 

6c05726